examples/evaluation/lora-evaluation.ipynb

```diff
@@ -0,0 +1,4712 @@
+      "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m111.1/111.1 kB\u001b[0m \u001b[31m8.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
+      "\u001b[?25hDownloading tqdm_multiprocess-0.0.11-py3-none-any.whl (9.8 kB)\n",
+      "Downloading DataProperty-1.0.1-py3-none-any.whl (27 kB)\n",
+      "Downloading mbstrdecoder-1.1.3-py3-none-any.whl (7.8 kB)\n",
+      "Downloading pathvalidate-3.2.1-py3-none-any.whl (23 kB)\n",
+      "Downloading tabledata-1.3.3-py3-none-any.whl (11 kB)\n",
+      "Downloading tcolorpy-0.1.6-py3-none-any.whl (8.1 kB)\n",
+      "Downloading typepy-1.3.2-py3-none-any.whl (31 kB)\n",
+      "Downloading portalocker-2.10.1-py3-none-any.whl (18 kB)\n",
+      "Downloading chardet-5.2.0-py3-none-any.whl (199 kB)\n",
+      "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m199.4/199.4 kB\u001b[0m \u001b[31m14.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
+      "\u001b[?25hBuilding wheels for collected packages: rouge-score, sqlitedict, word2number\n",
+      "  Building wheel for rouge-score (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for rouge-score: filename=rouge_score-0.1.2-py3-none-any.whl size=24934 sha256=85f16b4a2fc6fe3405432e65c5139fa405a5e2a13a87db4a1adb1238bf59bc6d\n",
+      "  Stored in directory: /root/.cache/pip/wheels/5f/dd/89/461065a73be61a532ff8599a28e9beef17985c9e9c31e541b4\n",
+      "  Building wheel for sqlitedict (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for sqlitedict: filename=sqlitedict-2.1.0-py3-none-any.whl size=16862 sha256=85bcc0eb8e2ca1631b031b42ffccf92084b1fea47b757fa1eae7c66eb49f5378\n",
+      "  Stored in directory: /root/.cache/pip/wheels/79/d6/e7/304e0e6cb2221022c26d8161f7c23cd4f259a9e41e8bbcfabd\n",
+      "  Building wheel for word2number (setup.py) ... \u001b[?25ldone\n",
+      "\u001b[?25h  Created wheel for word2number: filename=word2number-1.1-py3-none-any.whl size=5566 sha256=f42c7fa273de2bbf77d0d7fb73dbd9927bf373fe97823ae76025a655efb82520\n",
+      "  Stored in directory: /root/.cache/pip/wheels/84/ff/26/d3cfbd971e96c5aa3737ecfced81628830d7359b55fbb8ca3b\n",
+      "Successfully built rouge-score sqlitedict word2number\n",
+      "Installing collected packages: word2number, sqlitedict, tqdm-multiprocess, tcolorpy, portalocker, pathvalidate, jsonlines, chardet, sacrebleu, rouge-score, mbstrdecoder, typepy, peft, evaluate, DataProperty, tabledata, pytablewriter, lm_eval\n",
+      "Successfully installed DataProperty-1.0.1 chardet-5.2.0 evaluate-0.4.3 jsonlines-4.0.0 lm_eval-0.4.5 mbstrdecoder-1.1.3 pathvalidate-3.2.1 peft-0.13.2 portalocker-2.10.1 pytablewriter-1.2.0 rouge-score-0.1.2 sacrebleu-2.4.3 sqlitedict-2.1.0 tabledata-1.3.3 tcolorpy-0.1.6 tqdm-multiprocess-0.0.11 typepy-1.3.2 word2number-1.1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Install LM-Eval\n",
+    "!pip install lm_eval"
```

Comment:
Let's also install `datasets` and `evaluate`, as those are not PEFT requirements. Also, let's add `--quiet` to make the output less verbose.